src/components/HelpMark/HelpMark.tsx

```diff
@@ -13,33 +13,30 @@ import './HelpMark.scss';
 const b = block('help-mark');
 const ICON_SIZE = 16;
 
-export interface HelpMarkProps extends QAProps {
-    buttonProps?: React.ButtonHTMLAttributes<HTMLButtonElement>;
-    buttonRef?: React.RefObject<HTMLButtonElement>;
-    placement?: PopupPlacement;
-    className?: string;
+export interface HelpMarkProps extends QAProps, React.ButtonHTMLAttributes<HTMLButtonElement> {
+    popoverProps?: Omit<PopoverProps, 'children'>;
     children?: React.ReactNode;
 }
 
-export function HelpMark({
-    buttonRef,
-    buttonProps = {},
-    children,
-    className,
-    ...rest
-}: HelpMarkProps) {
+export const HelpMark = React.forwardRef<HTMLButtonElement, HelpMarkProps>(function HelpMark(
+    {children, qa, className, popoverProps, ...restProps},
+    ref,
+) {
     return (
-        <Popover {...rest} delayClosing={300} className={b(null, className)} content={children}>
-            {() => (
-                <button
-                    ref={buttonRef}
-                    type="button"
-                    {...buttonProps}
-                    className={b('button', buttonProps.className)}
-                >
-                    <Icon data={CircleQuestion} size={ICON_SIZE} />
-                </button>
-            )}
+        <Popover
+            content={<div className={b('popover')}>{children}</div>}
+            hasArrow
```

Comment:
what do you think if we add default for `delayClosing` to keep previous behaviour?

Reply:
discussed in pm, default will be set in `Popover`.

Reply:
New Popover has the default delay, not 300ms but 250ms